### PR TITLE
Improved vi_VN README Translation

### PR DESCRIPTION
**Pull request by NhanAZ**

### CODE REVIEW

**Comment by patricklodder** [comment]
Thanks!

Added "help wanted" - we could use a review from a native speaker.

File: README_zh_CN.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 </div>
 
-语言选择: [英文](./README.md) | 简体中文 | [PT](./README_pt_BR.md) | [FA](./README_fa_IR.md)
+语言选择: [英文](./README.md) | 简体中文 | [PT](./README_pt_BR.md) | [FA](./README_fa_IR.md)  | [VI](./README_vi_VN.md)
```
[the remark this answers is not in the document]
Comment (patricklodder) [reply README_zh_CN.md:14]: Good idea. Let's fix that separately from this, after this goes in (to not have to worry about merge conflicts)

**Review by mintodev — APPROVED** [review]
Nice Vietnamese translation. Some words are not translated because it will sound very impolite in Vietnamese.

**Review by patricklodder — APPROVED** [review]
ACK. I tested the links, translation was reviewed by @mintodev.

**Pull request merged**